.github/workflows/dev-environment-test.yml

```
@@ -0,0 +1,181 @@
+        if: always()
+        run: |
+          echo "🧹 Cleaning up..."
+          # Stop the development environment
+          if [ -n "$DEV_PID" ] && kill -0 $DEV_PID 2>/dev/null; then
+            echo "Stopping development environment (PID: $DEV_PID)"
+            kill $DEV_PID || true
+          fi
+          
+          # Stop docker-compose services
+          docker-compose down -v || true
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)